100of100Service/src/main/java/dev/coms4156/project/DatabaseConnection.java

```diff
@@ -9,26 +11,103 @@
+    } catch (SQLException e) {
+      e.printStackTrace();
+    }
   }
 
-  public List<Employee> getEmployees(int clientId) {
-    // TODO: Maybe some basic SQL query?
-    return null;
+  public List<Employee> getEmployees(int organizationId) {
+    List<Employee> employees = new ArrayList<>();
+    String query = "SELECT * FROM employees WHERE organization_id = ?";
+    try (PreparedStatement pstmt = connection.prepareStatement(query)) {
+      pstmt.setInt(1, organizationId);
+      ResultSet rs = pstmt.executeQuery();
+      while (rs.next()) {
+        Employee employee = new Employee(
+                null, // HRDatabaseFacade instance, passing null for now
+                rs.getInt("employee_id"),
+                rs.getString("name"),
+                rs.getDate("hire_date") // Assuming this field exists
+        );
+        employees.add(employee);
+      }
+    } catch (SQLException e) {
+      e.printStackTrace();
+    }
+    return employees;
   }
 
-  public List<Department> getDepartments(int clientId) {
-    // TODO: Maybe some basic SQL query?
-    return null;
+  public List<Department> getDepartments(int organizationId) {
+    List<Department> departments = new ArrayList<>();
+    String query = "SELECT * FROM departments WHERE organization_id = ?";
+    try (PreparedStatement pstmt = connection.prepareStatement(query)) {
+      pstmt.setInt(1, organizationId);
+      ResultSet rs = pstmt.executeQuery();
+      while (rs.next()) {
+        Department department = new Department(
+                null, // HRDatabaseFacade instance, passing null for now
+                rs.getInt("department_id"),
+                rs.getString("name"),
+                getEmployeesForDepartment(rs.getLong("department_id"))
+        );
+        departments.add(department);
+      }
+    } catch (SQLException e) {
+      e.printStackTrace();
+    }
+    return departments;
   }
 
-  public Organization getOrganization(int clientId) {
-    // TODO: Maybe some basic SQL query?
+  public Organization getOrganization(int organizationId) {
+    String query = "SELECT * FROM organizations WHERE organization_id = ?";
+    try (PreparedStatement pstmt = connection.prepareStatement(query)) {
+      pstmt.setInt(1, organizationId);
+      ResultSet rs = pstmt.executeQuery();
+      if (rs.next()) {
+        return new Organization(
+                null, // HRDatabaseFacade instance, passing null for now
+                rs.getLong("organization_id"),
+                rs.getString("name")
+        );
+      }
+    } catch (SQLException e) {
+      e.printStackTrace();
+    }
     return null;
   }
 
+  private List<Employee> getEmployeesForDepartment(long departmentId) {
+    List<Employee> employees = new ArrayList<>();
+    String query = "SELECT * FROM employees WHERE department_id = ?";
+
+    try (PreparedStatement pstmt = connection.prepareStatement(query)) {
+      pstmt.setLong(1, departmentId);
```

Comment:
Why did we set `departmentId` to be `long` in the database, but `int` in the Java's Department's constructor?